File: TP06_SanrioView/app/src/main/java/com/hsr2024/tp06_sanrioview/Item.java
package com.hsr2024.tp06_sanrioview;

//값을 넣어둘 나만의 리스트 만들기
public class Item {

    String name;
    String msg;
    int imgId;
    int tvnum=0;

    public Item(String name, String msg, int imgId) {
        this.name = name;
        this.msg = msg;
        this.imgId = imgId;
    }
}

File: Ex46KotlinRecyclerViewApp/app/src/main/java/com/hsr2024/ex46kotlinrecyclerviewapp/Item.kt
package com.hsr2024.ex46kotlinrecyclerviewapp

// 데이터 3개를 저장하는 용도의 클래스 - 일반 class와 다르게 .equals()를 사용하면 객체 주소가 아니라 값들을 비고해 줌.
// data를 쓰지 않으면 .equals()사용했을때 String이 아닌것들은 다 주소비교를 하기때문에... data class는 {}쓰지않는다.. 어차피 저장용도라서..
data class Item constructor(var title:String, var message:String, var imgId:Int) // {}안에 쓸게 없으면.. 생략가능..